dp-hello-world-controller/go.mod

```diff
@@ -1,26 +1,25 @@
 module github.com/ONSdigital/dp-hello-world-controller
 
-go 1.17
+go 1.18
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.105.0
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.167.0
 	github.com/ONSdigital/dp-component-test v0.7.0
 	github.com/ONSdigital/dp-healthcheck v1.3.0
-	github.com/ONSdigital/dp-net v1.4.1
-	github.com/ONSdigital/dp-renderer v1.21.0
+	github.com/ONSdigital/dp-net v1.5.0
+	github.com/ONSdigital/dp-renderer v1.46.1
 	github.com/ONSdigital/log.go/v2 v2.2.0
-	github.com/cucumber/godog v0.12.4
+	github.com/cucumber/godog v0.12.5
 	github.com/gorilla/mux v1.8.0
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/smartystreets/goconvey v1.7.2
```

Comment:
When I run the command to generate the controller, this dependency is still there?